R/count_teleconnections.R

```diff
@@ -123,6 +132,11 @@ count_watershed_teleconnections <- function(data_dir,
             }else{
               tc_n_landclasses = number_landclasses_ex_ag
             }
+        # TELECONNECTION - NUMBER OF FLOOD CONTROL DAMS WITHIN WATERSHED.
+        flood_control_sp <- sf::as_Spatial(flood_control_dams)
+
+        flood_control_sp[complete.cases(sp::over(flood_control_sp, watersheds_city)),] %>%
```

Comment:
What do `complete.cases` and `over` do here? Can we not subset simply using `flood_control_sp[watersheds_city, ]` ?